Tasks/Common/azure-arm-rest/azure-arm-endpoint.ts

```diff
@@ -23,7 +23,7 @@ export class AzureRMEndpoint {
         this.endpoint = null;
     }
 
-    public async getEndpoint(): Promise<AzureEndpoint> {
+    public async getEndpoint(useGraphActiveDirectoryResource: boolean = false): Promise<AzureEndpoint> {
```

Comment:
isn't this change already in master?

Reply:
It's there for azure-arm-rest-v2. I had to add for v1 as well, for Packer Build V0